src/test/java/pages/BasePage.java

```diff
@@ -12,9 +12,9 @@
 
       public class BasePage {
 
-        WebDriver driver;
-        WebDriverWait wait;
-        Actions actions;
+        static WebDriver driver;
+        static WebDriverWait wait;
+        public static Actions actions;
```

Comment:
I recommend: make the variables protected, because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

```
    protected Actions actions;
    protected WebDriverWait wait;
    protected WebDriver driver;
```